var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -27,6 +27,9 @@ class Paraview(CMakePackage, CudaPackage):
     tags = ["e4s"]
 
     version("master", branch="master", submodules=True)
+    version(
```

Comment:
Since the old RCs are not going to be released/have LTS, I think it is safe to replace the RC1 with RC2 here.

Reply:
This will need rebase shortly, this line will be updated shortly: https://github.com/spack/spack/pull/33486